### ISSUE TITLE
@joystream/types requires inconsistent @polkadot package versions

### ISSUE THREAD

**Issue by jfinkhaeuser** [Joystream/apps#199]
```json
  "dependencies": {
    "@polkadot/keyring": "^0.76.1",
    "@polkadot/types": "^0.77.0-beta.4",
    "@polkadot/util": "^0.76.1",
    "@polkadot/util-crypto": "^0.76.1",
    "typescript": "^3.4.5"
  },
```

that `@polkadot/types` beta version causes issues when trying to use the package together with `@polkadot/api`. All polkadot packages try to ensure that only a single version of themselves is in use at any time.

Relates to #196 in the sense that a new release should fix this.

**Comment by mnaamani** [comment]
I just thought to stick with the same versions as in the top package.json:

https://github.com/Joystream/apps/blob/joystream/package.json#L12

There is a corresponding `v0.77.0-beta.4` for the api.

**Comment by jfinkhaeuser** [comment]
See the discussion on #200, I think `0.76.1` is actually current - but I don't strongly care, as long as it's the same.

**Comment by jfinkhaeuser** [comment]
Fixed with #200
